### PR TITLE
Cleanup outdated PVC references

### PR DESCRIPTION
**Pull request by deekim**
→ corresponding [wh-cohorting-config PR](https://github.ibm.com/watson-health-cohorting/wh-cohorting-config/pull/2)

## References
- [WFFHCOHORT-850](https://jira.wh-sdlc.watson-health.ibm.com/browse/WFFHCOHORT-850)

### CODE REVIEW

File: cohort-evaluator-spark/pom.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
     <properties>
         <image.tag>us.icr.io/vpc-dev-cohort-rns/cohort-evaluator-spark:a-p3.4.3-tekton-20210907202747-a57c643820323a5c76ddb8392aa64c3eab27ba45</image.tag>
-        <config.volume>cohort-config</config.volume>
```
Comment (tabishop) [inline cohort-evaluator-spark/pom.xml:24]: We had discussed that we were going to remove this entire example section from the pom and the documentation.

**Review by tabishop — CHANGES_REQUESTED** [review]
There are also terraform scripts and override files in the config repo that need to get cleaned up which create/manage the pvc

**Comment by deekim** [comment]
@tabishop I added a  label to indicate that I'm still in the process of testing things.

I made the change you are suggesting in: https://github.ibm.com/watson-health-cohorting/wh-cohorting-config/pull/2
but haven't had a chance to run the `CD` toolchain to verify I didn't break anything.

**Review by msargentibm — APPROVED** [review]
LGTM

File: cohort-evaluator-hapi-fhir/pom.xml [hunk 2]
```diff
@@ -21,7 +21,7 @@
 
     <properties>
         <image.tag>us.icr.io/vpc-dev-cohort-rns/cohort-evaluator-spark:a-p3.4.3-tekton-20210907202747-a57c643820323a5c76ddb8392aa64c3eab27ba45</image.tag>
-        <config.volume>cohort-config</config.volume>
+        <config.bucket>cohort-config</config.bucket>
         <data.bucket>kwasny-test</data.bucket>
         <data.secret>spark-cos-secret</data.secret>
         <job.name>simple-job</job.name>
```
Comment (dkwasny-ibm) [inline cohort-evaluator-hapi-fhir/pom.xml:27]: I think I incorrectly copied these from the Spark POM when I created this one.
You deleted the properties from `cohort-spark-evaluator/pom.xml`, probably should here as well.

**Review by dkwasny-ibm — CHANGES_REQUESTED** [review]
Small issue caused by yours truly.

File: cohort-evaluator-hapi-fhir/pom.xml (hunk 2 above)
Comment (deekim) [reply cohort-evaluator-hapi-fhir/pom.xml]: _resolved_ in 960a596

**Review by jillrdoty — APPROVED** [review]

**Pull request merged**